CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Allowed Types of change: `Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `Security`
 
+## Unreleased
+
+## Fixed
+- SC-6931 fixed link names in Niedersachsen, Open and Thueringen to privacy and termsOfUse on homepage
+
+=======
```

Comment:
This ========= is redundant, right?

Reply:
I guess so yes, but was not sure if this is convention or not, so I kept it in. Shall I delete it?

Reply:
please delete it :)